scanner.py

```diff
@@ -2518,6 +2518,7 @@ def home_wait(self, home_end_time):
         res = [trsync.stop() for trsync in self._trsyncs]
         if any([r == etrsync.REASON_COMMS_TIMEOUT for r in res]):
             return -1.0
+            raise self.scanner.printer.command_error("Communication timeout during homing")
```

Comment:
This will never happen if you `return` 🤔
It should be marked as unreachable code by ruff?